src/components/homePage/Contributors.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div id="contributors">
+    <h1>Created with love in 🇨🇦 by</h1>
+    <div class="gallery">
+      <div class="parent" v-for="(member,index) in members" :key="member.name">
+        <a @click="open(index)" style="text-decoration: none;">
```

Comment:
It'd be **fantastic** if on desktop the dialogs opened up on hover instead of on click :))